sklearn/pipeline.py

```diff
@@ -615,6 +617,16 @@ class FeatureUnion(_BaseComposition, TransformerMixin):
         Multiplicative weights for features per transformer.
         Keys are transformer names, values the weights.
 
+    Examples
+    --------
+    >>> from sklearn.pipeline import FeatureUnion
+    >>> from sklearn.decomposition import PCA, TruncatedSVD
+    >>> union = FeatureUnion([("pca", PCA()),
```

Comment:
the example would be clearer with n_components=2 explicitly stated. Also, this could go in a separate PR.

Reply:
Changed, but also happy to leave this out (it's a change that was in the original PR of Andy)